src/Microsoft.AspNetCore.Mvc.Abstractions/project.json

```diff
@@ -33,17 +33,23 @@
       "version": "1.1.0-*",
       "type": "build"
     },
+    "Microsoft.Extensions.Localization.Abstractions": "1.1.0-*",
```

Comment:
@rynowak and I discussed this offline and the additional dependencies _are_ a concern for him. We also discussed alternatives and the best we can see is the `Func<string>` approach you were using around 7 iterations ago. That's still an ugly approach but it's better than introducing these dependencies. Sorry.